modules/cli-options/src/main/scala/scala/cli/commands/UninstallOptions.scala

```diff
@@ -0,0 +1,32 @@
+package scala.cli.commands
+
+import caseapp._
+import java.nio.file.Path
```

Comment:
NIT I'm surprised the organize imports scalafix rule didn't say anything, it should have required an empty line here:
```suggestion
import caseapp._

import java.nio.file.Path
```